### ISSUE TITLE
Still can't install code-d on OSX

### ISSUE THREAD

**Issue by p0nce** [Pure-D/code-d#95]
The installation process stays on this step:

`Installing into /Users/ponce/.vscode/extensions/webfreak.code-d-0.15.0/bin`

![capture d ecran 2016-12-23 a 01 47 20](https://cloud.githubusercontent.com/assets/1067485/21444789/cd3956d6-c8b1-11e6-8366-d7618ffbb913.png)

macOS Sierra 10.12.1, Visual Studio Code 1.8.1, dub 1.1.1, dmd v2.072.0-b2

**Comment by WebFreak001** [comment]
any proxy? does it maybe not have access to write into /Users/.../bin? Check the developer tools (ctrl-shift-p) for error messages

**Comment by p0nce** [comment]
In the console:
```
shell.ts:541 gitPath is not a function : TypeError: gitPath is not a function
    at startCompile (/Users/ponce/.vscode/extensions/webfreak.code-d-0.15.0/out/src/installer.js:266:30)
    at compileDependency (/Users/ponce/.vscode/extensions/webfreak.code-d-0.15.0/out/src/installer.js:293:9)
    at /Users/ponce/.vscode/extensions/webfreak.code-d-0.15.0/out/src/installer.js:170:9
    at FSReqWrap.cb [as oncomplete] (fs.js:257:19)
```

**Comment by WebFreak001** [comment]
uh that got fixed in #94 - gonna release a new version then
